docs/tutorials/preprocessing_tutorial.md

```diff
@@ -0,0 +1,174 @@
+# Preprocess Inputs using ElasticDL Preprocessing Layers
+
+The tutorials in this document aim to help users use ElasticDL preprocessing layers to transform model inputs.
+
+## Preprocessing Layers
+Generally, we cannot feed the raw data into the NN layer. So, ElasticDL provides the preprocessing layers to transform the raw data into numeric tensors. 
+The preprocessing layers allow users to include data preprocessing directly in their Keras model. Now, the preprocessing layers mainly aim to transform
+structured data (numeric data or string data) stored as a CSV file or a MaxCompute table and so on.  
```

Comment:
Remove `and so on` and replace with what is really supported if there are more supported data format.